content/code-security/dependabot/dependabot-version-updates/configuration-options-for-the-dependabot.yml-file.md

```diff
@@ -417,7 +417,7 @@ updates:
 
 ### `open-pull-requests-limit`
 
-By default, {% data variables.product.prodname_dependabot %} opens a maximum of five pull requests for version updates. Once there are five open pull requests, new requests are blocked until you merge or close some of the open requests, after which new pull requests can be opened on subsequent updates. Use `open-pull-requests-limit` to change this limit. This also provides a simple way to temporarily disable version updates for a package manager.
+By default, {% data variables.product.prodname_dependabot %} opens a maximum of five pull requests for version updates. Once there are five open pull requests from {% data variables.product.prodname_dependabot %}, {% data variables.product.prodname_dependabot %} will not open any new requests until some of those open requests are merged or closed. Use `open-pull-requests-limit` to change this limit. This also provides a simple way to temporarily disable version updates for a package manager.
```

Comment:
Thanks for working to address my earlier comment. However, I'm not sure this clarifies the issue I raised. In particular, from https://github.com/dependabot/dependabot-core/issues/3341#issuecomment-806000092, when the limit is reached, a new PR will be opened, but the old one will closed at the same time, keeping the number of open PRs at the limit. I don't think the "will not open any new requests" wording is consistent. Thoughts?

Reply:
I tried to get at that though in the wording of `"...are merged or closed."`. Since the old one _is_ closed, then that unlocks "open new pull requests"... so I don't see the conflict here? I do agree though that it requires parsing through the logic there to realize that.

Feel free to submit a docs PR if you have a suggestion for how to word it better, they're open to contributions from anyone, not just GH employees! 😄